### PR TITLE
Update config.toml 1.27 for 1.28 release

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Updating the supported releases for version 1.28. Supersedes #42516

/hold
/milestone 1.28

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
One of the reviews from the previous PR:

> If you rebase against main, you'll see that the fullversion parameter has already been removed.

Done, thanks!

**Comment by tengqm** [comment]
Well, you will need to rebase this to release-1.27 branch (to be created) anyway.

**Comment by Rishit-dagli** [comment]
Gotcha, tomorrow (one day before release), I should just try to create a new release-1.27 from dev-1.27 right?

**Comment by sftim** [comment]
You should set the base branch to _main_ and change to _release-1.27_ whenever that makes sense to do.

File: hugo.toml [hunk 1]
```diff
@@ -181,40 +180,40 @@ js = [
...
+githubbranch = "v1.28.0"
 docsbranch = "main"
 url = "https://kubernetes.io"
 
 [[params.versions]]
-fullversion = "v1.26.3"
+fullversion = "v1.27.4"
+version = "v1.27"
+githubbranch = "v1.27.4"
+docsbranch = "dev-1.27"
```
Comment (sftim) [inline hugo.toml:193]:
```suggestion
docsbranch = "release-1.27"
```

**Review by sftim — COMMENTED** [review]
These values are right. I would also mark the PR as draft for now.

**Comment by Rishit-dagli** [comment]
> These values are right. I would also mark the PR as draft for now.

Done

> 
You should set the base branch to main and change to release-1.27 whenever that makes sense to do.

Done, not resolving any conflicts now since the base will anyways be changed.

**Comment by Rishit-dagli** [comment]
> 
If you resolve conflicts for main, then when we cut release-1.27 from main, that resolution will stay valid.

Makes sense

**Comment by Rishit-dagli** [comment]
Changed base to `release-1.27`

**Review by sftim — COMMENTED** [review]
LGTM

Base branch might need to be recreated or updated though.

File: hugo.toml [hunk 2]
```diff
@@ -180,35 +180,36 @@ js = [
 ]
 
 [[params.versions]]
-version = "v1.27"
-githubbranch = "v1.27.0"
+fullversion = "v1.28.0"
```
Comment (sftim) [inline hugo.toml:183]: I don't expect to see a `fullversion` param.

**Review by sftim — COMMENTED** [review]
/approve

File: hugo.toml (hunk 2 above)
Comment (Rishit-dagli) [reply hugo.toml:183]: Ah, sorry again for missing this 😓

**Comment by Rishit-dagli** [comment]
Good to merge tomorrow with write access.

**Comment by sftim** [comment]
> Good to merge tomorrow with write access.

You should still wait for LGTM and approval, it's just that Prow won't automatically merge even with those in place.

/lgtm

If you squash this to 1 commit, that's even better @Rishit-dagli.

**Comment by Rishit-dagli** [comment]
Gotcha, I will also squash the commits.

File: hugo.toml [hunk 3]
```diff
@@ -138,11 +138,11 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
 
-latest = "v1.27"
+latest = "v1.28"
 
 version = "v1.27"
-githubbranch = "main"
-docsbranch = "main"
+githubbranch = "v1.27.4"
+docsbranch = "release-1.27"
 deprecated = false
```
Comment (divya-mohan0209) [inline hugo.toml:146]: Shouldn't deprecated=true here per [the handbook's example](https://github.com/kubernetes/website/pull/27455/files)?

Comment (reylejano) [reply hugo.toml:146]: @divya-mohan0209 is correct 
```suggestion
deprecated = true
```

Comment (Rishit-dagli) [reply hugo.toml:146]: I apologize for missing this earlier 😓

**Pull request merged**